dev/modulefiles/global_det/global_det_prep.sh

```diff
@@ -10,13 +10,22 @@ module load PrgEnv-intel/${PrgEnvintel_ver}
 module load intel/${intel_ver}
 module load ve/evs/${ve_evs_ver}
 module load gsl/${gsl_ver}
+module load netcdf/${netcdf_ver}
+module load cray-pals/${craypals_ver}
 module load prod_util/${prod_util_ver}
 module load libjpeg/${libjpeg_ver}
+module load libpng/${libpng_ver}
+module load zlib/${zlib_ver}
+module load jasper/${jasper_ver}
+module load udunits/${udunits_ver}
 module load grib_util/${grib_util_ver}
 module load wgrib2/${wgrib2_ver}
 module load cdo/${cdo_ver}
 module load udunits/${udunits_ver}
 module load nco/${nco_ver}
+module load met/${met_ver}
+module load metplus/${metplus_ver}
+module load bufr/${bufr_ver}
 
```

Comment:
Since you made changes to the modulefile here, did you make the appropriate changes in the ecf file as well?

Reply:
I did not, but will do that right now and then push those changes!

Reply:
Done!

Reply:
Excellent, thanks!